go.mod

```diff
@@ -1,11 +1,12 @@
-module service
+module %CUSTOM_PLUGIN_SERVICE_NAME%
```

Comment:
Hi, to made it work in ci, could you replace with new supported key: `mia_template_service_name_placeholder`?

Reply:
`%CUSTOM_PLUGIN_SERVICE_NAME%` replaced with `mia_template_service_name_placeholder` into the .gitignore and go.mod files